open_academy/security/open_academy_security.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+    <record id="session_group_manager" model="res.groups">
+        <field name="name">OpenAcademy / Manager</field>
+    </record>
+    <record id="course_rule_responsible_user" model="ir.rule">
+        <field name="name">Responsible user</field>
+        <field name="model_id" ref="model_course"/>
+        <field name="groups" eval="[(4, ref('session_group_manager'))]"/>
```

Comment:
Could you try this please?

For reference: https://github.com/Vauxoo/odoo/blob/15.0/odoo/fields.py#L2986

```suggestion
        <field name="groups" eval="[Command.link(ref('session_group_manager'))]"/>
```

Reply:
this works

Reply:
It's just a helper to construct the tuple, it's the same as it was before